react/src/lib/DatePicker/DatePickerDay/index.js

```diff
@@ -13,6 +13,18 @@ import moment from 'moment';
 class DatePickerDay extends React.Component {
   static displayName = 'DatePickerDay';
 
+  constructor(props) {
+    super(props);
+    this.dayButton = React.createRef();
+  }
+
+  componentDidUpdate = () => {
+    const { day } = this.props;
+    const { focus } = this.context;
```

Comment:
Will need to update this when #425 is merged

Reply:
You could branch off of your PR right?

Reply:
Or just include the commits together.